### DISCUSSION TITLE
Best way to annotate individual SV against merged SV set

### DISCUSSION THREAD

**Discussion by Phillip-a-richmond** [spiralgenetics/truvari#153] [General]
Hi Adam,

I'm wondering what the best way is to annotate+filter a single sample SV callset against the larger one generated. I'd be looking to remove polymorphic events and alignment/calling artifacts from a single sample with the goal of prioritizing rare SVs. 

Is there a vignette or example of you doing this? Would the proper workflow be:

1. Get set control SV VCFs, and your sample VCF (ideally from same caller)
2. BCFtools merge all together
3. Truavri collapse
4. truvari anno gtcnt
5. trucvari anno grpaf (sampleList.txt: controls with id, sample with id)
6. slivar filter on genotypes?

Thanks,
Phil

**Comment by ACEnglish** [comment]
If I'm understanding your goals correctly, that sounds like it would work. It'd also be reasonable to just use `truvari bench` with `--comp sample.vcf.gz` and `--base control.vcf.gz`. Then the subset of shared events between the two vcfs would be written to `tp-comp.vcf.gz` and `tp-base.vcf.gz` respectively.
